### PR TITLE
fix(webpack): make UMD build available on both browsers and Node.js

### PR DESCRIPTION
**Pull request by hanjeahwan**
Make UMD build available on both browsers and Node.js
## Description

Fix 'window is undefined' in node applications (SSR) 
## Types of changes

- [ x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: scripts/webpack.prod.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ module.exports = merge(baseConfig, {
     library: 'Scrollbar',
     libraryTarget: 'umd',
     libraryExport: 'default',
+    globalObject: 'this'
```
Comment (idiotWu) [inline scripts/webpack.prod.js]: Does this require a specific version of webpack? I'm using `4.40.2` in this project now.

Comment (hanjeahwan) [reply scripts/webpack.prod.js]: yup, this is one of the webpack 4 issue [https://github.com/webpack/webpack/issues/6677](https://github.com/webpack/webpack/issues/6677)

**Comment by hanjeahwan** [comment]
> The compiled files in `dist/` will be updated by running `npm run release`, you don't need to commit them in this pull request.

Deleted 😊

**Comment by hanjeahwan** [comment]
> Ah, don't delete the original files in `dist/` please 🤣.

Opsss,  revert done haha 😏

**Comment by idiotWu** [comment]
Thanks!

**Pull request merged**